.github/dependabot.yml

```
@@ -0,0 +1,26 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License
+---
+version: 2
+updates:
+  - package-ecosystem: "pip"
+    directories:
+      - "/clients/python"
+      - "/docker_tests"
+      - "/"
+    schedule:
+      interval: "daily"
```

Note: YAML only needs quotes if some templating/escaped are needed or if you want to force INT values being STR. I'd favor to remove quotes where not needed.
```suggestion
      interval: daily
```

---

Yeah agree, not sure will it work as per documentation its all in quotes ? and even dependaboat repo following same convention. 

We can try without quotes if needed :)

---

YAML is YAML. If quotes would be needed it is a bad YAML parser. In YAML it is always "less is more".

---

okay, have committed suggestion. if errors out we can revert. hopefully should be fine :)

---

Will be working. I bet a free berr... or free similar if it is not working next time we meet.

---

Hahaha  Done 🍻

---

Actually I do not like :beer: but I can offer some other :tropical_drink: as well when we meet.

---

Deal ... ahaha